src/connections/allContacts.js

```diff
@@ -104,3 +105,45 @@ export const withContactsMutations = withMutations(client => ({
 }))
 
 export default flow([withContacts, withContactsMutations])
+
+export function mergeContact(contact, attributes) {
+  // TODO: refactor to remove customizer from the mergeContact
+  const propertyName = {
+    email: 'address',
+    phone: 'number'
+  }
+
+  function customizer(objValue, srcValue, key) {
+    if (Array.isArray(objValue) || Array.isArray(srcValue)) {
+      return mergeArrayOnKey(objValue, srcValue, propertyName[key])
```

Comment:
What happens if `propertyName[key]` is `undefined`? Can that happen?

Reply:
I changed the implementation, can you check if it is all good.
If there is no strategie, the classical lodash's merge is used.